src/avt/VisWindow/Colleagues/VisWinRendering.C

```diff
@@ -1824,17 +1927,22 @@ VisWinRendering::BackgroundReadback(bool doViewportOnly)
 //    Brad Whitlock, Thu Sep 21 17:17:11 PDT 2017
 //    If we get 4 channel data, output 4 channel data.
 //
+//    Eric Brugger, Fri Oct 27 13:32:48 PDT 2023
```

Comment:
Does this work with VTK-8? I know we will be removing VTK-8 soon and it won't matter. Just wondering in the meantime if this will break VTK-8 usage.

Reply:
It doesn't. One of the vtk classes I use doesn't exist in VTK8. I will have to add VTK8 and VTK9 coding.

Reply:
I added VTK8 and VTK9 specific coding to VisWinRendering. I am currently running the regression suite with VTK8 and all the tests have passed in serial through roughly 50 tests, so it looks like the new coding is correct.